theme/Scripts/Panel_Playlist.js

```diff
@@ -2814,7 +2821,6 @@ function Header(x, y, w, h, idx, row_h_arg) {
         var clipImg = gdi.CreateImage(this.w, this.h);
         var grClip = clipImg.GetGraphics();
 
-        gr.FillSolidRect(this.x, this.y, this.w, this.h, g_pl_colors.background);
```

Comment:
The clipImg is drawn in this exact spot, so it seemed like this was left here in error.

Reply:
More like a left-over =)